src/config/2022.json

```diff
@@ -1008,6 +1008,16 @@
       "github": "foolip",
       "website": "https://foolip.org/"
     },
+    "decrecementofeliz": {
+      "name": "Pilar Mera",
+      "teams": [
+        "translators"
+      ],
+      "avatar_url": "1857745",
+      "github": "decrecementofeliz",
+      "twitter": "DecreceFeliz",
+      "website": "http://pi-comunicacion.com"
+    },
```

Comment:
@decrecementofeliz can you check you're happy with this?

Reply:
Hi, yes!

Reply:
Sounds great! Always happy to get more translations.